src/chat/chat_line_loading.rs

```diff
@@ -167,3 +176,76 @@ impl ChatLineLoadingRef {
         inner.timer = Timer::default();
     }
 }
+
+#[derive(Live, LiveHook, Widget)]
+pub struct AnswerLoading {
+    #[deref]
+    view: View,
+
+    #[rust]
+    couser: usize,
```

Comment:
Took me a second to understand this field (I think there's also a typo here), I think we should rename it so something more clear like `animation_frame` or something that more clearly describes that this is used to count the amount of dots.